### DISCUSSION TITLE
Adding menu item that has submenu to submenu causes submenu to not fully display

### DISCUSSION THREAD

**Discussion by Dante2202** [moudey/Shell#259] [Q&A]
I am using a registry tweak to have the firewall settings display in the context menu:
![1](https://github.com/moudey/Shell/assets/101029157/8efeddc8-d0f2-4330-a9c7-649ae99ef803)

I want to move this to the "Favorites" menu, I am able to get it inside the Favorites menu but the subfolder for the "Windows Defender Firewall" doesn't completely display:
![2](https://github.com/moudey/Shell/assets/101029157/af07983b-986c-4c8c-8e3c-3e03f95a955e)

I was using the below code in the "static" section on the shell.nss
item(find='Windows Defender Firewall' menu='Favorites')

**Comment by moudey** [comment]
There is an bug when moving items in this version, specifically when the names are similar, but it has been fixed in the [debug](http://nilesof.org/download/shell/debug.zip) version.  To avoid this issue as much as possible, match the search by putting double quotes.

```
item(find=' "Windows Defender Firewall" ' menu='Favorites')
```

**Comment by Dante2202** [comment]
Where do I add the static & dynamic sections to in the debug version?  I tried adding it to the following files with no success:
shell.nss
modifications.nss
new.nss
theme.nss

I am trying to add the following that I had working in the latest stable build:
```
static
	{	
		import 'imports/static.nss'
		item(where=this.title.length > 25 menu=title.more_options)
		item(find='NVIDIA Control Panel' vis=vis.remove)
		item(find='Share with Skype' vis=vis.remove)
		item(find='WinMerge' vis=vis.remove)
		item(find='Kill not responding tasks' position=-1)
		item(find='Display Settings' menu='Favorites')	
		item(find='Personalize' menu='Favorites')	
		item(find='MSI True Color' vis=vis.remove)
	}

	dynamic
	{
			
		menu(type='desktop|taskbar' title='Favorites' image=\uE0C8)
        {
                item(title='Registry editor' image cmd='regedit.exe')
                item(title='Paint' image cmd='C:\Program Files\Classic Paint\mspaint1.exe')
                item(title='Notepad++' image cmd='C:\Program Files (x86)\Notepad++\notepad++.exe')
		item(title='MSI True Color' image cmd='C:\Program Files\Portrait Displays\MSI True Color\MsiTrueColor.exe')
        }	
```

**Reply by moudey** [reply]
>Where do I add the static & dynamic sections to in the debug version? I tried adding it to the following files with no success:

The static, dynamic, var and images sections has been dropped.

Use  `modify` in "shell.nss" body instead of static item.

shell.nss
```
// static item
modify(find='Personalize' menu='Favorites')	

// dynamic item
menu(type='desktop|taskbar' title='Favorites' image=\uE0C8)
{
    item(title='Registry editor' image cmd='regedit.exe')
    ...
}	
```

**Comment by Dante2202** [comment]
Perfect, I can confirm debug version fixes my issue.  Thank you so much!

If I want to add a shortcut to the device manager, what would the command be to add to Shell?

![Untitled](https://github.com/moudey/Shell/assets/101029157/f18fcb36-d409-48ab-b2a7-a2e4417545fa)
